irnerio-mosaico/ontology/mmdio/README.md

```diff
@@ -0,0 +1,10 @@
+# Permanent Identifier for irnerio-mosaico ontology mmdio
+
+This identifier redirects to the mmdio ontology documentation for irnerio-mosaico.
+
+## Maintainer
+- [Your Name](faria.ferooz2@unibo.it)
+- [Your Organization](https://www.unibo.it/en)
```

Comment:
```suggestion
- [Faria Ferooz](faria.ferooz2@unibo.it)
- [Università di Bologna](https://www.unibo.it/en)
```

Reply:
@dgarijo I have updated the pull request to include my user ID and contact details in the README.md file